lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,9 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        "<video width=\"100%\" height=\"100%\" src=\"#{fname}\" controls> HTML5 video is not supported on this Browser.</video>"
```

Comment:
Yes, glad we come up with a simplified solution.  Yes we could do the same thing for the audio should be there any requests for it in the future.